includes/constants.php

```diff
@@ -3,13 +3,22 @@
 namespace Automattic\WP\Cron_Control;
 
 /**
- * Job queues
+ * Adjustable queue size and concurrency limits, to facilitate scaling
+ */
+$job_queue_size = defined( 'CRON_CONTROL_JOB_QUEUE_SIZE' ) && is_numeric( \CRON_CONTROL_JOB_QUEUE_SIZE ) ? absint( \CRON_CONTROL_JOB_QUEUE_SIZE ) : 10;
```

Comment:
A floor makes sense, good catch!

I'm not sure about a ceiling, as what's an appropriate upper limit largely depends on what's being done via cron and how cron is being executed. For now, something like 250 is probably safe. We can revisit should the need arise.